package.json

```diff
@@ -63,8 +63,9 @@
     "chai": "^1.9.1",
     "coveralls": "2.11.2",
     "dateformat": "^1.0.8",
-    "esprima-fb": "^10001.1.0-dev-harmony-fb",
     "eslint-tester": "^0.6.0",
+    "esprima-fb": "^10001.1.0-dev-harmony-fb",
```

Comment:
It is a devDependency. We need this for testing the pluggable parser feature, but I think any other parser should work too.